crates/ra_db/src/input.rs

```diff
@@ -107,6 +107,7 @@ impl CrateName {
 struct CrateData {
     file_id: FileId,
     edition: Edition,
+    declaration_name: Option<String>,
```

Comment:
```suggestion
    display_name: Option<String>,
```

Reply:
IE, the only purpose of this name is to be displayed to the user.

Reply:
Nice name for a name, thanks.